keybase/platform_windows_test.go

```diff
@@ -16,8 +16,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type testConfigWindows struct {
+	testConfigPlatform
+}
+
+func (c testConfigWindows) destinationPath() string {
+	return filepath.Join(os.Getenv("GOPATH"), "src/github.com/keybase/go-updater/windows/prompter/prompter.hta")
```

Comment:
This is the path to the production hta, which will be shipped - there isn't a test hta - where should it live?